src/split_ncvars/split_ncvars.pl.in

```diff
@@ -127,6 +127,10 @@ my $list_ncvars
     ? $ENV{LIST_NCVARS}
     : "$prefix/bin/list_ncvars.sh";
 
+unless (-f $list_ncvars) {
```

Comment:
A test scripot is required to ensure this line does what you expect.

Also, only testing if the file exists is not sufficient.  You do need to verify the command runs.  It would be easier to verify that the command ran successfully in the location where the command is run in this script than just trying to figure out if the command exits and is executable.

Reply:
Thank you very much. I that checking the exit status would be better than this check, and I'll do that in a separate PR.